example/node/index.js

```diff
@@ -40,43 +61,62 @@ async function run() {
   printOut(`Layer title:`, layer.title);
   printOut('Layer description:', layer.description);
 
-  if (!process.env.AUTH_TOKEN) {
-    printOut('Other examples need an auth token. Set env var AUTH_TOKEN and run again.', null);
+  if (!process.env.CLIENT_ID && !process.env.CLIENT_SECRET) {
+    printOut(
+      'Other examples need an auth token. Set env vars CLIENT_ID and CLIENT_SECRET and run again.',
+      null,
+    );
     return;
   }
 
+  let authToken;
+  const clientId = process.env.CLIENT_ID;
+  const clientSecret = process.env.CLIENT_SECRET;
+  printOut('Requesting auth token with client id and secret from env vars:', { clientId, clientSecret });
```

Comment:
I would suggest _not_ printing out the `clientSecret` - while users might be comfortable with putting it in `.env`, running the examples should not expose it. It is a secret after all. ;)

Reply:
I agree. Removed the `clientSecret`.